client/my-sites/comments/style.scss

```diff
@@ -23,3 +23,26 @@
 		padding: 10px 16px;
 	}
 }
+
+.comment-list__transition-enter {
+	opacity: 0.01;
+	visibility: hidden;
+
+	&.comment-list__transition-enter-active {
```

Comment:
Is it possible to un-nest this? I understand if we need the extra specificity here otherwise.